src/app/modules/home/__tests__/home.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import * as HomeController from 'src/app/modules/home/home.controller'
+
+import expressHandler from 'tests/unit/backend/helpers/jest-express'
+
+describe('home.controller', () => {
+  describe('home', () => {
+    const mockReq = expressHandler.mockRequest()
```

Comment:
`mockRes` (and probably `mockNext`) should be inside the test block itself, since the mock spies do not get reset before every test. if there were more usages for `mockRes`, the call times will not pass tests

Reply:
yeah, I know. i think the real question is whether such behavior should be relied upon - but easy enough to inline the code I suppose.